war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -271,5 +271,8 @@ THE SOFTWARE.
       <!-- See https://www.owasp.org/index.php/HttpOnly for the discussion of this topic in OWASP -->
       <http-only>true</http-only>
     </cookie-config>
+    <!-- Managed by WebAppMain.FORCE_SESSION_TRACKING_BY_COOKIE_PROP -->
+    <!-- This should be prevent the application server to use URL rewriting for session management -->
+    <!--<tracking-mode>COOKIE</tracking-mode>-->
```

Comment:
This comment isn't clear. Especially the part about commenting out the tracking mode.

If it is useful to leave a comment here, maybe say something like
```suggestion
    <!-- Tracking mode is managed by WebAppMain.FORCE_SESSION_TRACKING_BY_COOKIE_PROP -->
```

Reply:
Fine with that proposal :) The goal is to prevent people to do that in addition to the mechanism put in place ^^